apps/run/run_unix.go

```diff
@@ -1,3 +1,4 @@
+//go:build darwin || dragonfly || freebsd || linux || netbsd || openbsd
```

Comment:
As specified in the [Build constraints](https://pkg.go.dev/cmd/go#hdr-Build_constraints) doc:

> Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax.